ajax/libs/mavo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mavo",
+  "version": "0.2.0",
+  "description": "Create web applications with HTML and CSS",
+  "main": "mavo.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mavoweb/mavo.git"
+  },
+  "author": "Lea Verou",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mavoweb/mavo/issues"
+  },
+  "homepage": "https://mavo.io",
+  "autoupdate": {
```

Comment:
this doesn't look good to me. `npm*` keys shouldn't be in the `autoupdate` key. But we should have a proper CI to enforce that